Automattic/jetpack
# My Jetpack: Use Product Detail Table for Jetpack Protect

Fixes https://github.com/Automattic/jetpack/issues/29562

## Proposed changes:

* Introduces the recently added `ProductDetailTable` in the My Jetpack interstitial page for products that have `tiers` data.
* Adds related `tiers` data to the Protect product data in My Jetpack, to enable the pricing table for the Protect interstitial.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/29562

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Go to `/wp-admin/admin.php?page=my-jetpack#/add-protect`
  * Validate that the pricing table is displayed
  * Test the free plugin installation flow
  * Test the paid Scan plan purchase flow
* Go to the "add" page for other My Jetpack products, and check for regression.

## Screenshots

<img width="1210" alt="Screenshot 2023-04-28 at 11 08 39 AM" src="https://user-images.githubusercontent.com/10933065/235210488-6951c4d8-c1a2-4fa8-932d-de2c78bd569d.png">
